cluster/manifests/kube-proxy/daemonset.yaml

```diff
@@ -6,8 +6,6 @@ metadata:
   labels:
     application: kube-proxy
     version: v1.7.9_coreos.0
-  annotations:
-    rkt.alpha.kubernetes.io/stage1-name-override: coreos.com/rkt/stage1-fly
```

Comment:
This is only used when Kubernetes runs via `rkt` as container runtime?

Reply:
Yup: https://kubernetes.io/docs/getting-started-guides/rkt/